### ISSUE TITLE
FetchError: request to https://fakestoreapi.com/products failed, reason: connect ETIMEDOUT 104.21.37.32:443

### ISSUE THREAD

**Issue by 4mohdisa** [keikaavousi/fake-store-api#45]
This error happened while generating the page. Any console logs will be displayed in the terminal window.

getting this error from past 2 weeks 
![Uploading Screenshot 2022-02-12 045718.png…]()

**Comment by saurabhk06** [comment]
Yes, I am getting the same error. It's very frustrating as my app is getting crashed multiple times.

**Comment by keikaavousi** [comment]
Are you still have this issue?

**Comment by 4mohdisa** [comment]
> Do you still have this issue?

Yes

**Comment by saurabhk06** [comment]
Yes

**Comment by omeraladen** [comment]
Try To Change product to item
products.map((product) => (
              product.title
To:
products.map((item) => (
              item.title
